### PR TITLE
Sign up enabled

### PR DESCRIPTION
**Pull request by vijayakumarv-trika**
Sign up enabled

### CODE REVIEW

File: framework/commerce/types/signup.ts [hunk 1]
```diff
@@ -24,3 +24,8 @@ export type SignupSchema<T extends SignupTypes = SignupTypes> = {
     }
   }
 }
+
+export type SignupOperation = {
+  data: { result?: string }
+  variables: unknown
+}
```
Comment (GunaTrika) [inline framework/commerce/types/signup.ts:31]: why are we creatine new types?

Comment (GunaTrika) [reply framework/commerce/types/signup.ts:31]: Please delete this

File: framework/elasticpath/api/endpoints/signup/signup.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    email: email,
+    password: password
+  }
+  try {
+    let customerAPI = await Moltin.Customers.Create(customer)
+    return res.status(200).json(customerAPI);
+  } catch (error) {
+    let errorData = error.errors[0];
+    // Check if the email and password didn't match an existing account
+    if (errorData.status == 404) {
```
Comment (GunaTrika) [inline framework/elasticpath/api/endpoints/signup/signup.ts:37]: this condition is for login not found. please change is as per signup status codes

File: framework/elasticpath/auth/use-signup.tsx [hunk 3]
```diff
@@ -1,19 +1,46 @@
 import { useCallback } from 'react'
 import useCustomer from '../customer/use-customer'
 import { MutationHook } from '@commerce/utils/types'
+import { CommerceError } from '@commerce/utils/errors'
 import useSignup, { UseSignup } from '@commerce/auth/use-signup'
+import { constants } from 'buffer'
```
Comment (GunaTrika) [inline framework/elasticpath/auth/use-signup.tsx:6]: please remove unused imports

File: framework/commerce/types/signup.ts (hunk 1 above)
Comment (vijayakumarv-trika) [reply framework/commerce/types/signup.ts:31]: Without this application not allowing, we have the same operation for login as well but it is well written by nextjs, based on that only I've created

Comment (GunaTrika) [reply framework/commerce/types/signup.ts:31]: That is already written here right?
`framework/elasticpath/types/signup.ts `

**Pull request merged**